operations/iuf/workflows/management_rollout.md

```diff
@@ -155,105 +155,46 @@ Refer to that table and any corresponding product documents before continuing to
 
 1. Perform the NCN worker node upgrade. To upgrade worker nodes, follow the procedure in section [3.3 NCN worker nodes](#33-ncn-worker-nodes) and then return to this procedure to complete the next step.
 
-1. Upgrade `ncn-m001`.
+1. Upgrade `ncn-m001` with IUF.
 
-    1. Option 1 - Perform the NCN master node upgrade on `ncn-m001` without IUF
+    **`NOTE`** This subsection is mandatory only for Upgrade CSM and additional products with IUF.
```

Comment:
This Note can be removed as in management-nodes-rollout with CSM upgrade. Now we use CSM upgrade only with IUF. 
This Note was to skip the step while Upgrade CSM manually without IUF and use option 1
`1. Option 1 - Perform the NCN master node upgrade on `ncn-m001` without IUF`

Reply:
Good catch, fixed.